RESOURCES.md

```diff
@@ -0,0 +1,46 @@
+# Resources used by the Eiffel Community
+
+This file tries to track the resources used by the Eiffel Community,
+including the main contact person.
+To suggest the Eiffel Community to use a new resource,
+you can contribute by adding the resource to this file through a Pull Request.
+
+
+## Various resources
+
+| Resource                                                                           | Main Contact                                                                                              |
+|------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| [GitHub](https://github.com/eiffel-community)                                      | [TC](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members) |
+| [Slack](https://eiffel-workspace.slack.com/)                                       | Magnus Bäck                                                                                               |
+| [YouTube](https://www.youtube.com/@EiffelCommunity)                                | Emil Bäckmark                                                                                             |
+| LinkedIn                                                                           | ?                                                                                                         |

```

Comment:
```suggestion
| [LinkedIn](https://www.linkedin.com/company/40197226)             | [TC](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members)             |

```

Reply:
No, the link to LinkedIn is not yet there.

Reply:
Fixed in [d34c7a1](https://github.com/eiffel-community/community/pull/179/commits/d34c7a1210673aab0db7edbb0824b778cbf6223a)